db/schema.rb

```diff
@@ -125,7 +125,7 @@
   create_table "environment_variable_groups", id: :integer, force: :cascade do |t|
     t.string "name", null: false
     t.text "comment"
-    t.index ["name"], name: "index_environment_variable_groups_on_name", unique: true
+    t.index ["name"], name: "index_environment_variable_groups_on_name", unique: true, length: { name: 191 }
```

Comment:
I would undo all of the changes in here unrelated to your table, this file is a little hard to manage because samson supports three db types